hail/src/main/scala/is/hail/expr/ir/functions/GenotypeFunctions.scala

```diff
@@ -27,7 +27,7 @@ object GenotypeFunctions extends RegistryFunctions {
           tPL.isElementDefined(region, pl, i).mux(
             Code._empty,
             Code._fatal("PL cannot have missing elements.")),
-          pli := region.loadInt(tPL.loadElement(region, pl, i)),
+          pli := region.loadInt(tPL.loadElement(region, pl, len, i)),
```

Comment:
Why were you able to change the signature of loadElement? `len` doesn't look like it's used anywhere

Reply:
the signature without `len` will load the length to calculate how many missing bits there are, and therefore where the elements start. This signature is faster.